src/components/new-request/search.vue

```diff
@@ -242,46 +242,44 @@
 
       <!-- Restoration / Reinstatement flow -->
       <template v-else-if="isRestoration">
-        <BusinessLookupFetch ref="MyBusinessLookup" />
+        <BusinessLookupFetch />
         <SocietiesInfo
           v-if="isSocietyDisabled && showSocietiesInfo"
           :type="'restore'"
           :showDialog="showSocietiesInfo"
         />
-        <template v-else>
-          <CompanyType v-if="getSearchBusiness && isBcRestorable && isSupportedRestoration(getEntityTypeCd)" />
-          <Jurisdiction
-            v-if="isSelectedXproAndRestorable"
+        <CompanyType v-if="getSearchBusiness && isBcRestorable" />
```

Comment:
The other part of the fix was to remove "is supported restoration" from this component. That logic is used elsewhere (for determining what buttons to show).